packages/suite/src/utils/suite/logsUtils.ts

```diff
@@ -180,12 +192,25 @@ export const getApplicationInfo = (state: AppState) => ({
     torOnionLinks: state.suite.settings.torOnionLinks,
     labeling: state.metadata.enabled ? state.metadata.provider?.type || 'missing-provider' : '',
     analytics: state.analytics.enabled,
+    instanceId: hideSensitiveInfo ? REDACTED_REPLACEMENT : state.analytics.instanceId,
+    sessionId: hideSensitiveInfo ? REDACTED_REPLACEMENT : state.analytics.sessionId,
     transport: state.suite.transport?.type,
     transportVersion: state.suite.transport?.version,
     wallets: state.devices.length,
-    devices: [...new Set(state.devices.map(device => device.id))].length,
     rememberedStandardWallets: state.devices.filter(d => d.remember && d.useEmptyPassphrase).length,
     rememberedHiddenWallets: state.devices.filter(d => d.remember && !d.useEmptyPassphrase).length,
     enabledNetworks: state.wallet.settings.enabledNetworks,
     customBackends: getCustomBackends(state.wallet.blockchain).map(({ coin }) => coin),
+    devices: getPhysicalDeviceUniqueIds(state.devices)
+        .map(id => state.devices.find(device => device.id === id)) // filter unique devices
+        .map((device: TrezorDevice) => ({
+            id: hideSensitiveInfo ? REDACTED_REPLACEMENT : device.id,
+            label: hideSensitiveInfo ? REDACTED_REPLACEMENT : device.label,
```

Comment:
what could be the benefit from knowing device label? I guess `id` is enough?

Reply:
My idea was that it would be easier for user if support tell him - your device XYZ bla bla (in case he has it renamed)

Reply:
ok then, makes sense 👌